### PR TITLE
Run encrichments on rolling or stdview

### PR DESCRIPTION
**Pull request by lindsey-w**
## Description

This change allows use to use different aggregation levels when running the enrichment process. 

## Testing

Tested with rolling (with start/end, with default) as well as with a stdview.

### CODE REVIEW

File: wintappy/datautils/rawutil.py [hunk 1]
```diff
@@ -17,6 +17,8 @@ def init_db(dataset=None, agg_level="rolling", database=":memory:", lookups=""):
     Initialize an in memory db instance and configure with our custom sql.
     """
     con = duckdb.connect(database=database)
+    # set caching dir to a temp directory location
+    con.execute(f"SET temp_directory = '{tempfile.mkdtemp()}'")
```
Comment (lindsey-w) [inline wintappy/datautils/rawutil.py:21]: we 'ok' with including this here?

Comment (grantjllnl) [reply wintappy/datautils/rawutil.py:21]: Yes, that should make it more tolerate of high memory usage cases.

**Review by grantjllnl — COMMENTED** [review]
This all looks good. Go ahead merge!

**Review by grantjllnl — APPROVED** [review]
Approved

**Pull request merged**